host-controller/src/main/java/org/jboss/as/host/controller/resources/StoppedServerResource.java

```diff
@@ -67,4 +94,19 @@ public void registerOperations(final ManagementResourceRegistration resourceRegi
         resourceRegistration.registerOperationHandler(CompositeOperationHandler.INTERNAL_DEFINITION, CompositeOperationHandler.INSTANCE);
     }
 
+    class LaunchTypeHandler implements OperationStepHandler {
```

Comment:
This can be private and static. Static removes a ref to StoppedServerResource.this from memory.